code/modules/events/radiation.dm

```diff
@@ -122,8 +122,8 @@
 				animate_flash_color_fill_inherit(T,"#FFDD00",1,5)
 				return
 		animate_flash_color_fill_inherit(T,"#0084ff",1,5)
-		for (var/atom/A in T.contents)
-			A.changeStatus("n_radiation", (rad_strength) SECONDS, 3)
+		for (var/mob/A in T.contents)
+			A.take_radiation_dose(rad_strength/5 SIEVERTS)
```

Comment:
this ends up being weaker than the non-neutron one with current numbers (2Sv vs 2.5Sv).
Could make it actually irradiate stuff (w/ component etc) in the vicinity in the future?